src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -937,4 +937,26 @@
   <data name="ColorPicker_Editor.Text" xml:space="preserve">
     <value>Editor</value>
   </data>
+  <data name="PowerToys_Run_PositionAppearance_GroupSettings.Text" xml:space="preserve">
+    <value>Position &amp; appearance</value>
+  </data>
+  <data name="PowerLauncher_MonitorToDisplayOn.Header" xml:space="preserve">
+    <value>Specify a monitor index</value>
+  </data>
+  <data name="PowerToysRun_PositionHeader.Text" xml:space="preserve">
+    <value>Show PowerToys Run on</value>
+  </data>
+  <data name="Radio_Position_Cursor.Content" xml:space="preserve">
+    <value>Monitor with mouse cursor</value>
+  </data>
+  <data name="Radio_Position_Focus.Content" xml:space="preserve">
+    <value>Monitor with focused window</value>
+  </data>
+  <data name="Radio_Position_Primary.Content" xml:space="preserve">
+    <value>Primary monitor</value>
+  </data>
+  <data name="Windows_Display_Settings.Text" xml:space="preserve">
+    <value>Windows display settings</value>
+    <comment>Windows refers to the Operating system</comment>
+  </data>

```

Comment:
Let's make names consistent. Use `Run_` as a prefix for PowerToys Run page.

Reply:
PowerLauncher_MonitorToDisplayOn.Header is not used anymore

Reply:
Windows_Display_Settings is not used anymore

Reply:
Thanks; I've removed the unused rows. On the `Radio_Position` entries my thinking is they're universal so can be used on other settings pages; a bit like `Radio_Theme`. Happy to change to `Run_`, let me know :)

Reply:
Let's change all 5 labels to the prefix `Run_`. It is very unlikely that we would have to use `Radio_Position`s in other modules. IMO label should not be reused if we want to reuse a label that means that we can move it into separate reusable control(possibly with logic).